### PR TITLE
docs(bodiless-documentation): Fix migration documentation

### PR DESCRIPTION
**Pull request by hvanyo**
## Changes
Minor doc updates

### CODE REVIEW

File: packages/bodiless-documentation/doc/Development/OtherResources/npm_commands.md [hunk 1]
```diff
@@ -12,7 +12,6 @@ NPM commands, that are defined in the root `package.json` file and their purpose
 | `copyright`            | Adds copyright notice header to sll source files |
 | `fix`                  | Runs an eslint scan of the entire project and fixes all auto-fixable problems |
 | `fix:target`           | Runs an eslint scan of a particular file or directory and fixes all auto-fixable problems |
-| `fresh`                | Clean everything, remove node_modules and then run `bootstrap` |
```
Comment (beliayeu) [inline packages/bodiless-documentation/doc/Development/OtherResources/npm_commands.md]: @hvanyo, we still have fresh command in root [package.json](https://github.com/johnsonandjohnson/Bodiless-JS/blob/master/package.json#L21). so I believe we need to keep description of this command

File: packages/bodiless-documentation/doc/Development/OtherResources/Lessons.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Issues
 
 ### rimraf Is Not Available on Windows
-* **Issue**: Errors with rimraf when the command `npm run fresh` is executed.
+* **Issue**: Errors with rimraf when the command `npm run setup` is executed.
```
Comment (wodenx) [inline packages/bodiless-documentation/doc/Development/OtherResources/Lessons.md]: Why is this necessary?  Isn't rimraf a project dependency?

Comment (hvanyo) [reply packages/bodiless-documentation/doc/Development/OtherResources/Lessons.md]: On windows rimraf has to be installed global to have permissions to work.  I was just switching fresh/setup but realized that you only removed fresh from platform.sh.  But our documentation refers to run setup.  Let me do this -- combine all the darn window workaround together..

File: packages/bodiless-migration-tool/doc/Technical/MigrationTool.md [hunk 3]
```diff
@@ -33,40 +33,39 @@ The tool is given URL that will crawl the existing site finding all pages, gener
...
-```sh
+1. Let's create a site to migrate the files to.
 
-npm run fresh
+    The full directions can be found [here](/About/GettingStarted?id=creating-a-new-site)
+    We recommend this path is outside the repository root.  In the code example, we used `../NEW_MIGRATED_SITE`.
 
-```
+    ```sh
+    npm run ci
```
Comment (beliayeu) [inline packages/bodiless-migration-tool/doc/Technical/MigrationTool.md:53]: npm run ci -> npm ci

File: packages/bodiless-migration-tool/doc/Technical/MigrationTool.md [hunk 4]
```diff
@@ -33,40 +33,39 @@ The tool is given URL that will crawl the existing site finding all pages, gener
...
+    npm run new ../NEW_MIGRATED_SITE
+    cd ../NEW_MIGRATED_SITE
+    npm install
```
Comment (beliayeu) [inline packages/bodiless-migration-tool/doc/Technical/MigrationTool.md:56]: I could be wrong, but as far as I can see install is already included in 'npm run new ../NEW_MIGRATED_SITE'

File: packages/bodiless-migration-tool/doc/Technical/MigrationTool.md (hunk 3 above)
Comment (hvanyo) [reply packages/bodiless-migration-tool/doc/Technical/MigrationTool.md]: Fixed in https://github.com/johnsonandjohnson/Bodiless-JS/pull/143/commits/9bd0417dc698a10bca0d86066c1295236b4ea0d1

File: packages/bodiless-migration-tool/doc/Technical/MigrationTool.md (hunk 4 above)
Comment (hvanyo) [reply packages/bodiless-migration-tool/doc/Technical/MigrationTool.md]: Fixed in https://github.com/johnsonandjohnson/Bodiless-JS/pull/143/commits/9bd0417dc698a10bca0d86066c1295236b4ea0d1
